how-to/misc/bootstrap-highly-available-juju-controller-on-top-of-a-lxd-cluster.rst

```diff
@@ -16,6 +16,11 @@ Requirements
 
 You will need:
 
+* one dedicated physical network with:
+  
+  * an unlimited access to the Internet
+  * DHCP server configured to serve LXD containers
```

Comment:
Using a generic DHCP server contradicts with a warning we had in the "old" doc.

> Caution: Any change in IP address of the local host will be detrimental to the deployment. A virtual host will generally have a more stable address.

Is it the case still? (I assume yes)

Also, would be good to address my question in the original issue

> If it's from a DHCP server, where is the DHCP server with what kind of configurations? We assume 3 physical hosts only here.

Reply:
You are right regarding the dynamic allocation. Given that it's currently impossible to bootstrap Juju controllers with static IP addresses in such a setup LXD containers would need to be manually re-configured after bootstrap to use the same IPs as originally allocated but static. I pushed an update listing this deficiency as a warning.

However, I disagree with your commentary on the DHCP server itself. Since every environment is different, product documentation needs to remain neutral here. We could potentially point users to the [isc-dhcp-server documentation](https://ubuntu.com/server/docs/how-to-install-and-configure-isc-dhcp-server) here, but I see no point in doing that as DHCP might already be running in their network. Thus, it fair to list it as a requirement instead.